sentry_sdk/integrations/stdlib.py

```diff
@@ -185,7 +185,9 @@ def getresponse(self, *args, **kwargs):
                     response_data = rv.read()
                     # once we've read() the body it can't be read() again by the
                     # app; save it so that it can be accessed again
-                    rv.read = io.BytesIO(response_data).read
+                    saved_response = io.BytesIO(response_data)
+                    rv.read = saved_response.read
+                    rv.fp = saved_response
```

Comment:
what is fp? please update the PR description so there's an explanation in the commit

Reply:
updated

Reply:
thx, this probably affected other integrations too then